backend.pm

```diff
@@ -309,6 +327,10 @@ sub qemu_start
 	run_string($self, "$self->{'root_password'}");
 	wait_prompt($self);
 	run_string($self, "export PS1=\$ ");
+
+    if (defined($self->{'qemu_virtfs'})) {
```

Comment:
I think there are some tab vs space things. 
What do you think of https://github.com/metan-ucw/runltp-ng/pull/6

Reply:
I could also setup perltidy, percritic and travis, @metan-ucw do you wanna have it?

Reply:
@cfconrad sounds good, is there a common coding style for a perl or do we have to choose any?

Reply:
The two tools I know from os-autoinst is perltidy and perlcritic.
The first one makes some coding style checks. Like linewrapping, space between func and arguments and so forth. 
The later is for deeper inspection and try to avoid common bad practices.
My proposal is, I would give it a deeper look and come up with best fitting configs. Then we make a Makefile and a `check` target to lunch these tools.

Reply:
Sounds good, also we may as well set up a git pre-commit hook.

Reply:
yeah, I think it should be minimal though, just make sure we either use tabs or spaces. perltidy does some very questionable stuff in os-autoinst.

Reply:
lets discuss here: https://github.com/metan-ucw/runltp-ng/pull/7